packages/DraftEditor/src/interfaces/index.tsx

```diff
@@ -0,0 +1,175 @@
+  ariaActiveDescendantID?: string;
+
+  value?: string;
+
+  customStyleFn?: (
+    style: DraftInlineStyle,
+    block: ContentBlock
+  ) => DraftStyleMap;
+  utils: {
+    uniqueId: (element: any) => string;
+    ErrorMessage: FunctionComponent<IErrorMessage>;
+    classNames: (className: string | object, classNames?: object) => string;
+  };
+
+  onChange?: (event: {
+    event: { target: { name: string; value: string } };
+  }) => void;
+  error?: boolean;
+  validationMessage: string | JSX.Element;
+
+  handleReturn?(
+    e: SyntheticKeyboardEvent,
+    editorState: EditorState
+  ): DraftHandleValue;
+
+  handleKeyCommand?(
+    command: DraftEditorCommand,
+    editorState: EditorState,
+    eventTimeStamp: number
+  ): DraftHandleValue;
+
+  handleBeforeInput?(
+    chars: string,
+    editorState: EditorState,
+    eventTimeStamp: number
+  ): DraftHandleValue;
+
+  handlePastedText?(
+    text: string,
+    html: string | undefined,
+    editorState: EditorState
+  ): DraftHandleValue;
+
+  handleDroppedFiles?(
+    selection: SelectionState,
+    files: Blob[]
+  ): DraftHandleValue;
+
+  handleDrop?(
+    selection: SelectionState,
+    dataTransfer: object,
+    isInternal: DraftDragType
+  ): DraftHandleValue;
+
+  onBlur?(e: SyntheticEvent): void;
+  onFocus?(e: SyntheticEvent): void;
+  onTab?(e: SyntheticKeyboardEvent): void;
+  blockRendererFn?(block: ContentBlock): any;
+  blockStyleFn?(block: ContentBlock): string;
+  onEscape?(e: SyntheticKeyboardEvent): void;
+  onUpArrow?(e: SyntheticKeyboardEvent): void;
+  onLeftArrow?(e: SyntheticKeyboardEvent): void;
+  onDownArrow?(e: SyntheticKeyboardEvent): void;
+  onRightArrow?(e: SyntheticKeyboardEvent): void;
+  handlePastedFiles?(files: Blob[]): DraftHandleValue;
+  keyBindingFn?(e: SyntheticKeyboardEvent): DraftEditorCommand | null;
+}
+
+export interface IInlineImageControlsProps {
+  editorState: EditorState;
+  previewImages: IImage[];
+  onToggle: (newEditorState: EditorState, entityKey: string) => EditorState;
+}
+
+export interface ILinkControlProps {
+  editorState: EditorState;
+  unSelectedText: string;
+  onToggle: (
+    newEditorState: EditorState,
+    selection: SelectionState,
+    entityKey: string
+  ) => void;
+}
+
+export interface IInlineTypes {
+  label: string;
+  style: string;
+}
+
+export interface IInlineControlsProps {
+  editorState: EditorState;
+  onToggle: (blockType: DraftBlockType) => void;
+}
+
+export interface IStyleButtonProps {
+  onToggle: (style: string) => void;
+  style: string;
+  active: boolean;
+  label: any;
+  utils: {
+    uniqueId: (element: any) => string;
+    classNames: (className: string | object, classNames?: object) => string;
+  };
+}
+
+export interface ILinkProps {
+  contentState: ContentState;
+  children: JSX.Element | JSX.Element[];
+  entityKey: string;
+}
+
+export interface IPreviewIages {
+  previewImages: IImage[];
+  isSelected: (src: string) => boolean;
+  addSelectedImage: (image: IImage) => void;
+}
```

Comment:
I would do a common export at the end of the file, easier to manage what is being exported.